5-AccessControl/2-call-api-groups/AppCreationScripts/sample.json

```diff
@@ -88,8 +91,8 @@
           "value": "$tenantId"
         },
         {
-          "key": "Enter_the_Web_Api_Scope_here",
-          "value": "service.Scope"
+          "key": "Enter_the_Web_Api_Application_Id_Here",
```

Comment:
When the scrips create the groups, there's a way to update the group IDs in the configuration files. See the example: https://github.com/Azure-Samples/ms-identity-javascript-nodejs-tutorial/blob/268dea1f65c87cb884712f3a0711e7b8c7769095/4-AccessControl/2-security-groups/AppCreationScripts/sample.json#L65